res/controllers/Denon-MC7000-scripts.js

```diff
@@ -305,10 +406,9 @@ MC7000.padModeFlip = function(channel, control, value, status, group) {
     MC7000.PADModeSampler[deckNumber] = false;
     MC7000.PADModeVelSamp[deckNumber] = false;
     MC7000.PADModePitch[deckNumber] = false;
-
     // switch off PAD illumination
     for (var i = 0; i < 8; i++) {
-        midi.sendShortMsg(0x94 + deckOffset, 0x1C + i, MC7000.padColor.alloff);
+        midi.sendShortMsg(0x94 + deckOffset, 0x14 + i, MC7000.padColor.alloff);
```

Comment:
to explain a bit better, when deactivating with shift, the button address changes to 0x1C, otherwise without shift button it is 0x14. That's why the change from 0x1C to 0x14 was required, initialisation without shift button. This is for pad mode flip, currently not used, but should be corrected anyway.

Reply:
Ok, so the controller does the shift functionality by-itself. So now the padModeFlip happens on the shift-layer instead the unshifted one? But below it still uses 0x14 to control the unshifted led.